torchreid/integration/sc/parameters.py

```diff
@@ -77,6 +77,13 @@ class __LearningParameters(ParameterGroup):
             affects_outcome_of=ModelLifecycle.TRAINING
         )
 
+        enable_early_stopping = configurable_boolean(
+            default_value=True,
+            header="Enable adaptive early stopping of the training",
+            description="Smart exit from training when accuracy isn't changed or decreased for several epochs.",
```

Comment:
I'd say early here instead of smart, one could imagine different definitions of smartness here.